x/programs/rust/wasmlanche_sdk/src/store/mod.rs

```diff
@@ -102,7 +120,7 @@ impl From<i64> for Context {
     }
 }
 
-fn store_key_value<T>(ctx: &Context, key_bytes: Vec<u8>, value: &T) -> Result<(), StorageError>
+fn store_key_value<T>(ctx: Context, key_bytes: &[u8], value: &T) -> Result<(), StorageError>
```

Comment:
This was an interesting one... passing in a `Vec<u8>` here assures that the memory is deallocated after this function exits, which *might* be desired. However, I did find one place where we were calling `to_vec` to pass in a value to `store_key_value`, so taking in a slice requires one less allocation. 

*I think* this is better, but I'm open to an argument for the contrary.

Reply:
passed in a Vec<u8> since we don't need the memory after, but if its faster I'm all for it.